README.md

```diff
@@ -160,6 +160,32 @@ pluginNodePolyfill({
 });
 ```
 
+### only
```

Comment:
How about naming this option to `include`? This is more consistent with `exclude`

Same as https://github.com/davidmyersdev/vite-plugin-node-polyfills?tab=readme-ov-file#customizable-when-you-need-it

Reply:
Suire, updated.